polling_stations/templates/postcode_view.html

```diff
@@ -60,12 +60,15 @@
       {% endif %}
 
     </div>
+    {% if multiple_elections %}
+       {% include "fragments/multiple_elections.html" %}
+    {% endif %}
 
     {% include "fragments/register_to_vote.html" %}
     {% include "fragments/you_dont_need_poll_card.html" %}
 
-    {% if we_know_where_you_should_vote and election_explainers and has_election and not error %}
-      {% include "election_explainers.html" %}
+    {% if council.electoral_services_phone_numbers and council.electoral_services_email %}
```

Comment:
How does this work in Scotland and Northern Ireland?

Reply:
Also, what's the system for taking this info off the site? Do we have a reminder? Or should we try to write code to turn it off?

Reply:
Scotland and Northern Ireland are good points. I'll include `has_election` in the if. Also we shouldn't be showing it in NI ever, as we don't store council specific info for NI. I think it's OK for Scotland because we do have council specific electoral services info.

Reply:
It's mentioned in [the wiki](https://github.com/DemocracyClub/UK-Polling-Stations/wiki/Maintenance-tasks#post-election) and in the `election playbook` google sheet. I think that's ok for now.

Reply:
And for Scotland, are we sure that it's the local council who hire for polling stations? I assume so, but would be good to double check!

Reply:
@pmk01 do you know the answer to this :point_up:?

Reply:
Pretty sure it's the council, or at any rate the info will be found on the council website eg.: https://www.highland.gov.uk/info/799/elections_and_voting/603/training_and_employment_at_elections